### ISSUE TITLE
Submit file doesn't work, can't run LiSa

### ISSUE THREAD

**Issue by tsarpaul** [danieluhricek/LiSa#11]
I'm getting this error on worker, the other contains don't throw any exceptions:

```
worker_1    | Traceback (most recent call last):
worker_1    |   File "/usr/local/lib/python3.6/site-packages/billiard/pool.py", line 1796, in safe_apply_callback
worker_1    |     fun(*args, **kwargs)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/celery/worker/request.py", line 371, in on_failure
worker_1    |     store_result=self.store_errors,
worker_1    |   File "/usr/local/lib/python3.6/site-packages/celery/backends/base.py", line 160, in mark_as_failure
worker_1    |     traceback=traceback, request=request)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/celery/backends/base.py", line 342, in store_result
worker_1    |     request=request, **kwargs)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/celery/backends/database/__init__.py", line 53, in _inner
worker_1    |     return fun(*args, **kwargs)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/celery/backends/database/__init__.py", line 105, in _store_result
worker_1    |     session = self.ResultSession()
worker_1    |   File "/usr/local/lib/python3.6/site-packages/celery/backends/database/__init__.py", line 99, in ResultSession
worker_1    |     **self.engine_options)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/celery/backends/database/session.py", line 59, in session_factory
worker_1    |     self.prepare_models(engine)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/celery/backends/database/session.py", line 54, in prepare_models
worker_1    |     ResultModelBase.metadata.create_all(engine)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 4287, in create_all
worker_1    |     ddl.SchemaGenerator, self, checkfirst=checkfirst, tables=tables
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2032, in _run_visitor
worker_1    |     with self._optional_conn_ctx_manager(connection) as conn:
worker_1    |   File "/usr/local/lib/python3.6/contextlib.py", line 81, in __enter__
worker_1    |     return next(self.gen)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2024, in _optional_conn_ctx_manager
worker_1    |     with self._contextual_connect() as conn:
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2226, in _contextual_connect
worker_1    |     self._wrap_pool_connect(self.pool.connect, None),
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2266, in _wrap_pool_connect
worker_1    |     e, dialect, self
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1536, in _handle_dbapi_exception_noconnection
worker_1    |     util.raise_from_cause(sqlalchemy_exception, exc_info)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 383, in raise_from_cause
worker_1    |     reraise(type(exception), exception, tb=exc_tb, cause=cause)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 128, in reraise
worker_1    |     raise value.with_traceback(tb)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2262, in _wrap_pool_connect
worker_1    |     return fn()
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 354, in connect
worker_1    |     return _ConnectionFairy._checkout(self)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 751, in _checkout
worker_1    |     fairy = _ConnectionRecord.checkout(pool)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 483, in checkout
worker_1    |     rec = pool._do_get()
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/pool/impl.py", line 237, in _do_get
worker_1    |     return self._create_connection()
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 299, in _create_connection
worker_1    |     return _ConnectionRecord(self)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 428, in __init__
worker_1    |     self.__connect(first_connect_check=True)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 630, in __connect
worker_1    |     connection = pool._invoke_creator(self)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
worker_1    |     return dialect.connect(*cargs, **cparams)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 453, in connect
worker_1    |     return self.dbapi.connect(*cargs, **cparams)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/pymysql/__init__.py", line 94, in Connect
worker_1    |     return Connection(*args, **kwargs)
worker_1    |   File "/usr/local/lib/python3.6/site-packages/pymysql/connections.py", line 325, in __init__
worker_1    |     self.connect()
worker_1    |   File "/usr/local/lib/python3.6/site-packages/pymysql/connections.py", line 630, in connect
worker_1    |     raise exc
worker_1    | sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on '172.42.0.14' ([Errno 111] Connection refused)")
worker_1    | (Background on this error at: http://sqlalche.me/e/e3q8)
```

**Comment by jrespeto** [comment]
do you still need help?

**Comment by tsarpaul** [comment]
No, I don't need LiSa any more, but I couldn't use it at all due to this error.
A colleague of mine had the same issue, you might want to keep this issue open..

**Comment by danielpoliakov** [comment]
Hi, should be fixed by 3e57bc9
